packages/chrome/demo/~patterns/stories/DarkStory.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { StoryFn } from '@storybook/react';
+import { DefaultTheme } from 'styled-components';
+import { DEFAULT_THEME, IGardenTheme, ThemeProvider, getColor } from '@zendeskgarden/react-theming';
+import { Chrome, Body, Content } from '@zendeskgarden/react-chrome';
+import { Nav } from './components/Nav';
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+
+interface IArgs {
+  test: boolean;
+}
+
+export const DarkStory: StoryFn<IArgs> = args => {
+  const theme = (parentTheme: DefaultTheme) =>
+    ({
+      ...parentTheme,
+      colors: {
+        ...parentTheme.colors,
+        base: 'dark',
+        background: getColor('neutralHue', 900, parentTheme),
+        foreground: getColor('neutralHue', 200, parentTheme),
+        primaryHue: '#50a1e1'
+      }
+    }) as IGardenTheme;
```

Comment:
For the POC, this is the main theming setup for dark mode – affecting a major portion of the UI as expected.